widget/embedded/src/components/SwapDetailsAlerts/SwapDetailsAlerts.styles.ts

```diff
@@ -5,3 +5,27 @@ export const Alerts = styled('div', {
   flexDirection: 'column',
   gap: '$10',
 });
+
+export const ActionIcon = styled('div', {
+  transition: 'transform 0.5s ease-in-out',
+  variants: {
+    rotated: {
+      true: {
+        transform: 'rotate(180deg)',
+      },
+    },
+  },
+});
+
+export const AlertFooter = styled('div', {
+  transition: 'max-height 0.5s ease-in-out',
+  maxHeight: '0px',
+  overflow: 'hidden',
+  variants: {
+    open: {
+      true: {
+        maxHeight: '$40',
+      },
+    },
+  },
+});
```

Comment:
Can we try using the collapsible component here? If that's not an option, let's at least keep the transition duration consistent for uniformity : 0.5 s -> 300ms

Reply:
I had tried to use it but I couldn't. Just changed the transition.